examples/todo/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Check out https://hub.docker.com/_/node to select a new base image
+FROM node:10-slim
```

Comment:
@nabdelgadir I might miss some details, wondering...what is the docker setup file used for?

Reply:
By default, a Dockerfile is generated. Since we're showing the users to accept all defaults, this file is added. It originated from [this discussion](https://github.com/strongloop/loopback-next/pull/4412#issuecomment-575303326).

Reply:
ah I see, good point, thank you for the explanation